homeassistant/components/hive/sensor.py

```diff
@@ -126,6 +126,9 @@ async def async_update(self) -> None:
         """Update all Node data from Hive."""
         await self.hive.session.updateData(self.device)
         self.device = await self.hive.sensor.getSensor(self.device)
-        self._attr_native_value = self.entity_description.fn(
-            self.device["status"]["state"]
-        )
+        if "status" in self.device:
+            self._attr_native_value = self.entity_description.fn(
+                self.device["status"]["state"]
+            )
+        else:
+            self._attr_native_value = None
```

Comment:
```suggestion
        self._attr_native_value = self.entity_description.fn(
            self.device.get("status", {}).get("state")
        )
```
I think this works as well and looks cleaner

Reply:
I've made that change.
Thanks